### PR TITLE
docker-{load,diff}: add French translation

### PR DESCRIPTION
**Pull request by ghost**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/docker-load.md
Comment (gutjuri) [inline pages.fr/common/docker-load.md:4]:
```suggestion
> Plus d'informations: <https://docs.docker.com/engine/reference/commandline/load>.
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thank you for your contribution! Are the descriptions written in the third person singular present indicative tense? (see https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules)

Pardon me asking, I don't know French :D

**Review by kbdharun — APPROVED** [review]
~LGTM,~ Thanks for your contribution.

Edit. LGTM, after the suggestions are addressed. (I have requested a review from another maintainer who is a native French speaker)

File: pages.fr/common/docker-load.md
Comment (kbdharun) [reply pages.fr/common/docker-load.md:4]: This should probably be removed from the English page too, or else the test will fail. Will do it now.

File: pages.fr/common/docker-diff.md
Comment (kbdharun) [inline pages.fr/common/docker-diff.md:10]:
```suggestion
> Inspecte les changements apportés aux fichiers ou dossiers sur le système de fichiers d'un conteneur.
> Plus d'informations: <https://docs.docker.com/engine/reference/commandline/diff>.

- Inspecte les changements apportés à un conteneur depuis sa création:

`docker diff {{conteneur}}`

- Affiche l'aide:
```

File: pages.fr/common/docker-load.md
Comment (kbdharun) [inline pages.fr/common/docker-load.md:14]:
```suggestion
> Charge des images docker depuis des fichiers ou `stdin`.
> Plus d'informations: <https://docs.docker.com/engine/reference/commandline/load/>.

- Charge une image docker depuis `stdin`:

`docker load < {{chemin/vers/fichier_image.tar}}`

- Charge une image docker depuis un fichier spécifique:

`docker load --input {{chemin/vers/fichier_image.tar}}`

- Charge une image docker depuis un fichier spécifique en mode silencieux:
```

File: pages.fr/common/docker-load.md
Comment (nicokosi) [inline pages.fr/common/docker-load.md:3]: :innocent: 
```suggestion
> Charger des images Docker depuis des fichiers ou `stdin`.
```

File: pages.fr/common/docker-load.md
Comment (nicokosi) [inline pages.fr/common/docker-load.md:4]:
```suggestion
> Plus d'informations : <https://docs.docker.com/engine/reference/commandline/load/>.
```

File: pages.fr/common/docker-load.md
Comment (nicokosi) [inline pages.fr/common/docker-load.md:6]:
```suggestion
- Charger une image Docker depuis `stdin` :
```

File: pages.fr/common/docker-load.md
Comment (kbdharun) [reply pages.fr/common/docker-load.md:10]: Thanks for the remainder, I keep forgetting to review spacing in French PRs 😅 . Updated my previous suggestions too.

**Pull request merged**
